src/css/parallelcoordinates.css

```diff
@@ -25,3 +25,8 @@
 .nvd3 .axis text {
     text-shadow: 0 1px 0 #fff;
 }
+
+.nvd3 .hover  {
+    fill-opacity: 1;
+	stroke-width: 3px;
+}
```

Comment:
This will apply to way more than parallel coordinates, you need to have a class before .hover that's specific to this chart.